templates/cassiopeia/scss/vendor/choicesjs/choices.scss

```diff
@@ -74,30 +79,36 @@
   }
 
   .choices__inner {
-    padding: $custom-select-padding-y ($custom-select-padding-x + $custom-select-indicator-padding) $custom-select-padding-y $custom-select-padding-x;
-    background: url("../../../images/select-bg.svg") no-repeat right center / $custom-select-bg-size;
-    background-color: $custom-select-bg;
+    padding: $custom-select-padding-y $custom-select-indicator-padding $custom-select-padding-y $custom-select-padding-x;
+    background: $custom-select-bg url("../../../images/select-bg.svg") no-repeat right center / $custom-select-bg-size;
 
     [dir=rtl] & {
-      padding: $custom-select-padding-y $custom-select-padding-x $custom-select-padding-y ($custom-select-padding-x + $custom-select-indicator-padding);
-      background: url("../../../images/select-bg-rtl.svg") no-repeat left center / $custom-select-bg-size;
-      background-color: $custom-select-bg;
+      padding: $custom-select-padding-y $custom-select-padding-x $custom-select-padding-y $custom-select-indicator-padding;
+      background: $custom-select-bg url("../../../images/select-bg-rtl.svg") no-repeat left center / $custom-select-bg-size;
     }
   }
 }
-.choices__list--multiple .choices__item {
-  font-size: inherit;
-  font-weight: 500;
-  color: $white;
-  background-color: var(--info);
-  border: 1px solid $white;
-  border-radius: $border-radius;
-
-  &.is-highlighted {
+
+.choices__list--single {
+  padding-right: .25em;
```

Comment:
Do we need an RTL case for this for padding-left?

Reply:
yes - 
![grafik](https://user-images.githubusercontent.com/1035262/99914217-2dff0300-2cfc-11eb-8537-9af80609195a.png)